src/simulate.js

```diff
@@ -70,14 +99,12 @@ function parseStartPage(projectRoot) {
     }
 
     return 'index.html'; // default uri
-}
+};
 
+module.exports = simulate;
+module.exports.launchBrowser = launchBrowser;
+module.exports.launchServer = launchServer;
+module.exports.closeServer = closeServer;
```

Comment:
Is there still a need to expose any of these externally? Previously `launchBrowser` and `launchServer` were exposed so taco-simulate could call them.